components/authorization/api-model/src/commonMain/kotlin/Roles.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2025 The ORT Server Authors (See <https://github.com/eclipse-apoapsis/ort-server/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.eclipse.apoapsis.ortserver.components.authorization.api
+
+import kotlinx.serialization.Serializable
+
+/** An enum for the available organization roles. */
+@Serializable
+enum class OrganizationRole { READER, WRITER, ADMIN }
```

Comment:
I am not sure whether these different enum classes actually provide some value. For the REST API, the user passes a string; so, there is no improvement in type-safety. Whether this string is converted directly to a model role or via the indirection through the mappings, does not make a big difference, does it?